src/Temporalio/Worker/TemporalWorkerOptions.cs

```diff
@@ -36,6 +36,44 @@ public TemporalWorkerOptions()
         /// <param name="taskQueue">Task queue for the worker.</param>
         public TemporalWorkerOptions(string taskQueue) => TaskQueue = taskQueue;
 
+        /// <summary>
+        /// Event for when a workflow task is starting. This should only be used for very advanced
```

Comment:
Since it's experimental and we can change it, maybe call it `ActivationStarting`?

Reply:
The goal is task tracking even if the implementation isn't there yet. But I think the experimental API should reflect the goal instead of the current reality. I would have waited on this PR another but wanted to let users see the goal early.